### PR TITLE
refactor: /stats -> /about

### PR DESCRIPTION
**Pull request by Nciklol**

### CODE REVIEW

File: services/backend/src/commands/System/about.ts [hunk 1]
```diff
@@ -6,14 +7,19 @@ import {
...
 
-export class StatsCommand extends Command {
+export class AboutCommand extends Command {
   public constructor(context: Command.Context, options: Command.Options) {
     super(context, {
       ...options,
-      name: "stats",
-      description: "View an overview of my system.",
+      name: "about",
+      description: "View some info about evie!",
```
Comment (twisttaan) [inline services/backend/src/commands/System/about.ts:22]:
```suggestion
      description: "View information about Evie and useful links.",
```

File: services/backend/src/commands/System/about.ts [hunk 2]
```diff
@@ -43,18 +51,46 @@ export class StatsCommand extends Command {
       (line) => `${Emojis.bulletPoint} ${line}`
     );
 
+    const commandStats =
+      await this.container.client.prisma.commandStats.findMany();
+
+    const formatted = commandStats
+      .sort((a, b) => b.uses - a.uses)
+      .map((c, i) => `${i + 1}. ${c.name}: ${c.uses}`)
+      .filter((_, i) => i < 5);
```
Comment (twisttaan) [inline services/backend/src/commands/System/about.ts:60]: This is really messy, [ninja code](https://javascript.info/ninja-code) isn't really the style we're going for.

File: services/backend/src/commands/System/about.ts [hunk 3]
```diff
@@ -43,18 +51,46 @@ export class StatsCommand extends Command {
...
         name: "Growth (all shards)",
         value: growth.join("\n"),
       },
+      {
+        name: "Top 5 Commands:",
+        value: formatted.join("\n"),
+      },
     ]);
 
-    embed.setFooter({
```
Comment (twisttaan) [inline services/backend/src/commands/System/about.ts:53]: Why remove this? This is the only way the user can see what shard their guild has. Useful for debugging in the future.

**Review by twisttaan — CHANGES_REQUESTED** [review]
Looks good to me, just act on the suggestions 🚀

File: services/backend/src/commands/System/about.ts (hunk 2 above)
Comment (twisttaan) [reply services/backend/src/commands/System/about.ts:60]: It would also be nice to make a util to parse numbers so 1000 would become 1,000.

Comment (megatank58) [reply services/backend/src/commands/System/about.ts:60]: Can't we simply use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

Comment (twisttaan) [reply services/backend/src/commands/System/about.ts:60]: > Can't we simply use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

Yep, sounds good.

**Review by twisttaan — APPROVED** [review]
LGTM

**Pull request merged**